ncm-filesystems/src/main/pan/components/filesystems/schema.pan

```diff
@@ -8,10 +8,14 @@ include 'quattor/schema';
 include 'quattor/blockdevices';
 include 'quattor/filesystems';
 
+@documentation{
+when manage_blockdevs is false, filesystems does same as fstab
```

Comment:
with an alias, you run fstab component, but either with the fstab.pm or the filesystems.pm code.

that way the code is not in the same module, but the relation is much clearer.